### DISCUSSION TITLE
Kafka without ZooKeeper

### DISCUSSION THREAD

**Discussion by d3bt3ch** [strimzi/strimzi-kafka-operator#8168] [General]
Hi,

I can see that `Kafka without ZooKeeper` is on the roadmap. When can we expect it to be released for production use?

**Comment by scholzj** [comment]
There are previous discussions here and on Slack which you can probably check out. In short, it needs to be ready in Kafka first -> the upgrade is not production ready, JBOD support is missing, SCRAM-SHA authentication is missing, etc. All of these are key features which need to be completed.
